client/tests/webdriver/baseSpecs/insights.spec.js

```diff
@@ -1,8 +1,9 @@
 import { expect } from "chai"
+import { PENDING_ASSESSMENTS_BY_POSITION } from "../../../src/pages/insights/Show"
```

Comment:
Tests should in principle not import from the application itself. You could simply export it from `insights.page` and change the import below to:
```
import Insights, { INSIGHTS, PENDING_ASSESSMENTS_BY_POSITION } from "../pages/insights.page"
```